src/services/openAI.service.ts

```diff
@@ -1,23 +1,24 @@
+import { CLIArguments } from "../types";
+import { fileName } from "../types/renameFiles";
 import { cleanFileNameExtension, httpRequest } from "../utils";
 import dotenv from 'dotenv';
 dotenv.config();
 const OPENAI_API_KEY = process.env.OPENAI_API_KEY;
 
-
-export const generateFileName = async (content: string): Promise<string> => {
+export const generateFileName = async (content: string, baseFileName:fileName, args: CLIArguments): Promise<string> => {
```

Comment:
Will their be a case when OpenAI responses with `null` or with no response of a miss from API endpoint. We need to handle this either in this or in a separate ticket.

Reply:
We will be handling that in near future.